pkg/lexer/token.go

```diff
@@ -87,6 +87,9 @@ const (
 	WHILE  // while
 	BREAK  // break
 	END    // end
+
+	PKG    // package keyword, reserved for later use
+	IMPORT // import keyword, reserved for later use
```

Comment:
every token has a test in `TestSingleToken()`. I guess these should be added too.

Reply:
Good point! Addressed.